plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -852,6 +901,24 @@ function MosaicMenu:_recalculateDimen()
             height = corner_mark_size,
         }
     end
+
+    -- Create or replace progress_widget if needed
+    local progress_bar_width = self.item_width * 0.60;
+
+    if not progress_widget or progress_widget.width ~= progress_bar_width then
+        -- if progress_widget then
+        --     progress_widget:free()
+        -- end
```

Comment:
Why comment out this ?

Reply:
What happened is, if I try to resize emulator, this part actually throws an error saying `free` is nil. So, I assumed `free` is unavailable.

Reply:
Oh right :) it seems ProgressWidget doesn't have and need any :free().

Reply:
So, you can remove this commented out stuff.